schemas/com.snowplowanalytics.snowplow.enrichments/pii_pseudonymizer_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for PII pseudonymization enrichment",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.enrichments",
+    "name": "pii_pseudonymizer_config",
```

Comment:
Can we change the name (and thus filenames and table name etc) to `pii_enrichment_config`. pseudonymizer is just one behavior of the enrichment.